.github/workflows/linux_riscv.yml

```diff
@@ -41,7 +41,7 @@ jobs:
 
   Build:
     needs: Smart_CI
-    timeout-minutes: 150
+    timeout-minutes: 200
```

Comment:
I suppose the problem is not with limits, the job just hangs for some reason.
It happened why Andrey B played with new 32gb runners

Reply:
That seems to be true, the job timed out at 200 minutes.